client/sender.go

```diff
@@ -93,7 +93,7 @@ func newSender(u *url.URL, ctx *base.Context, retryOptions retry.Options, stoppe
 // in that case.
 func SendWrapped(sender Sender, ctx context.Context, args roachpb.Request) (roachpb.Response, error) {
 	if ctx == nil {
```

Comment:
Maybe, but removing it implies changing hundreds of lines that call this function with a `nil` context. For a convenience method I don't think it's worth it.